app/models/container_definition.rb

```diff
@@ -9,6 +9,7 @@ class ContainerDefinition < ApplicationRecord
 
   def disconnect_inv
     _log.info "Disconnecting Container definition [#{name}] id [#{id}]"
+    self.container.disconnect_inv
```

Comment:
We really shouldn't do this, as the save_inventory should eventually hit the container removal.